README.md

```diff
@@ -70,6 +73,7 @@ Start by including the base set of variables:
 @import "node_modules/@spectrum-css/tokens/dist/index.css";
 
 /*
+  TODO
```

Comment:
Is this TODO intentional? This comment block is part of the code snippet.

Reply:
Good catch! I'll remove